modules/series-service-impl/src/main/java/org/opencastproject/series/impl/SeriesServiceImpl.java

```diff
@@ -198,7 +198,7 @@ public DublinCoreCatalog updateSeries(DublinCoreCatalog dc) throws SeriesExcepti
 
         logger.debug("Updating series {}", id);
         // update API index
-        updateSeriesMetadataInIndex(id, elasticsearchIndex, dublinCore);
+        updateSeriesMetadataInIndex(id, index, dublinCore);
```

Comment:
I guess we could kick the index out of these methods as well, but it's not strictly necessary.